libs/client/editor-lite/src/lib/editor-lite.component.html

```diff
@@ -265,3 +288,41 @@
         <rmx-icon name="h-3"></rmx-icon>Heading 3
     </button>
 </mat-menu>
+
+<mat-menu #insertImageMenu="matMenu">
+    <form [formGroup]="addImage" (ngSubmit)="insertImage()">
+        <input
+            [formControlName]="'imageUrl'"
+            [type]="'text'"
+            [placeholder]="'Paste image URL'"
+            class="border-t-0 border-l-0 border-r-0 border-gray-600 dark:border-white"
+            style="background: var(--background);"
+            (click)="$event.stopPropagation()"
+        >
+        <div class="flex justify-center items-center mt-1">
+            <button type="submit">
+                <rmx-icon name="add-box-line"></rmx-icon>
+                <span>Insert</span>
+            </button>
+        </div>
+    </form>
+</mat-menu>
+
+<mat-menu #insertMediaMenu="matMenu">
+    <form [formGroup]="addMedia" (ngSubmit)="insertMedia()">
+        <input
+            [formControlName]="'mediaUrl'"
+            [type]="'text'"
+            [placeholder]="'Paste media URL'"
```

Comment:
Since this only allows YouTube links, should it say "Paste YouTube URL" to make it a little clearer?